templates/rust/macros.tera

```diff
@@ -178,11 +213,32 @@ pub mod {{cluster_mod}} {
     #[allow(unused_imports)]
     use crate::common::{*};
     {% for register_name,reg in cluster.registers -%}
-    {{self::register_struct(reg=reg)}}
+    {{self::register_struct(reg_name=reg.name,reg=reg)}}
     {% endfor -%}
     {% for cluster_name,cluster in cluster.clusters -%}
     {{self::cluster_struct(cluster=cluster) }}
     {% endfor -%}
 }
 {%- endif -%} {# if not cluster.is_derived_from #}
 {%- endmacro -%}
+
+
+{# Unroll the cluster register for aurix csfr #}
```

Comment:
Is it possible to move this macro and the cluster_register_core_func in aurix_core.tera to improve the separation between standard template and only aurix template ?

Reply:
Done